collective-project-backend/Controllers/InternshipController.cs

```diff
@@ -77,13 +80,30 @@ public IActionResult AddInternship(Internship internship)
 
         }
     
-
+        [Route("{id:int}")]
         [HttpPut]
```

Comment:
doar compania poate face update internship, please adauga atribut de authorize   [Authorize(Roles="Company")] (vezi in internshipController)